src/phaser-game/scenes/FortNerf.ts

```diff
@@ -1,28 +1,44 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disable @typescript-eslint/no-unused-vars */
-/* eslint-disable @typescript-eslint/no-explicit-any */
-/* eslint-disable @typescript-eslint/no-shadow */
 import Phaser from 'phaser';
```

Comment:
note: there's a merge conflict somewhere in this file that needs to be addressed. not sure where tho.